### PR TITLE
Added 'migrate stripe <command>'

### PR DESCRIPTION
**Pull request by sagzy**
closes https://github.com/TryGhost/Product/issues/3621

- attaches `@tryghost/mg-stripe` package to the main `migrate` command. The new `copy`, `confirm` and `revert` commands can be called using `migrate stripe <command>` syntax
- deletes duplicated CLI setup inside the `mg-stripe` package

### CODE REVIEW

File: packages/migrate/commands/stripe.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+};
+
+export default {
+    id,
+    group,
+    flags,
+    desc,
+    setup
+};
+
```
Comment (SimonBackx) [inline packages/migrate/commands/stripe.js:57]: I think this is duplicate of `packages/mg-stripe/src/lib/command.ts` and can normally be replaced by just importing the command like this: https://github.com/TryGhost/migrate/blob/2aff3a03471135e449719bb2ff7dc4d835141f19/packages/mg-stripe/src/bin/cli.ts#L9-L10

Comment (SimonBackx) [reply packages/migrate/commands/stripe.js:57]: Hmmm... I saw you moved it 🤔 I'm not sure, I think it was nice that it was self contained

Comment (sagzy) [reply packages/migrate/commands/stripe.js:57]: Yeah fair point. Let me reiterate on that tomorrow

Comment (sagzy) [reply packages/migrate/commands/stripe.js]: Reverted it back to how it was before — you're right, it's nice that it's self-contained, easier to move around later on if needed.

**Pull request merged**